tests/reference/asr-array_01_decl-39cf894.stderr

```diff
@@ -0,0 +1,11 @@
+warning: The module 'numpy' located in $DIR/src/bin/../runtime/lpython_intrinsic_numpy.py cannot be loaded
+ --> tests/../integration_tests/array_01_decl.py:2:1
+  |
+2 | from numpy import empty, int16, int32, int64, float32, float64, complex64, complex128
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ imported here
+
+semantic error: Decorator: vertorize is not supported
+   --> $DIR/src/bin/../runtime/lpython_intrinsic_numpy.py:421:2
+    |
+421 | @vertorize
+    |  ^^^^^^^^^ 
```

Comment:
I think this is not needed. This seems to be a reference output of an old version of the changes in this PR.

You can remove this by using:

(First ensure that your important changes (if any) are safe (either committed or in a new branch)):
```bash
rm -rf tests/reference/*
(After executing the above, you might need to press y here if there is any prompt saying "are you sure you want to delete all these files")
```
Now just generate the reference tests again.
```bash
./run_tests.py
```

Reply:
Make sense. Committed in https://github.com/lcompilers/lpython/pull/2316/commits/e2bfbff7893496e45fe7eb793d90c6f68c2f7178. Thanks!